### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Wingman4** [community/community#137191] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Someone is digging through my files and disabling my devices

**Comment by Wingman4** [comment]
I've practically wiped my phone and hid things in my secure folder.

**Comment by Wingman4** [comment]
if this in one the developers. they need to never supervise my account again

**Comment by Wingman4** [comment]
![Screenshot (14)](https://github.com/user-attachments/assets/828428f1-45b2-4d42-a324-5eeda8c6000a)
Found this

**Comment by Wingman4** [comment]
I'm being blocked from filing a police report

**Comment by LiteBrite82** [comment]
Hi @Wingman4 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
